jaas/src/main/java/com/griddynamics/jagger/jaas/service/DynamicDataService.java

```diff
@@ -38,22 +37,25 @@
     
     private static final Logger LOGGER = LoggerFactory.getLogger(DynamicDataService.class);
     
-    private final ExecutorService destroyerService =
-            Executors.newSingleThreadExecutor(r -> new Thread("DynamicDataServiceDestoyer"));
+    private final ExecutorService destroyerService = Executors.newSingleThreadExecutor(r -> {
```

Comment:
Yes. I can. But it is even better to extend existing default implementation, isn't it?